CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ### Version 9.6
 * Feign builder now supports flag `doNotCloseAfterDecode` to support lazy iteration of responses.
+* Introduces `JacksonIteratorDecoder` in `feign-jackson` which provides a closeable iterator for lazy processing json arrays. This decoder has to be used with `Feign.Builder.doNotCloseAfterDecode()` option disabled. Built-in iterator auto closes the `Response` when it reached json array end or failed to parse stream.
```

Comment:
wording tip. Changelog is highlight not how-to. Your first sentence is fine assuming javadoc or readme elsewhere says the how-to. Same thing below